helix-term/src/commands.rs

```diff
@@ -69,13 +69,15 @@ use grep_searcher::{sinks, BinaryDetection, SearcherBuilder};
 use ignore::{DirEntry, WalkBuilder, WalkState};
 use tokio_stream::wrappers::UnboundedReceiverStream;
 
+pub type KeyCallback = Box<dyn FnOnce(&mut Context, KeyEvent)>;
```

Comment:
very minor naming nit but I think this could be clearer as `OnNextKeyCallback` to match the fields it is used on

Reply:
fixed